### PR TITLE
Add new fields to enable cache in the repositories

### PR DESCRIPTION
**Pull request by kaduartur**
### Description
To decrease the number of requests to check if a repository has a new version, we are implementing a caching approach. Two new fields have been added to the repo structure which are the `LatestVersion` and `Cache`.

### How to verify it
This PR should not change the behavior of any command in the rit

### Changelog
Add new fields to Repo struct

### CODE REVIEW

File: pkg/formula/repo/adder.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (ad AddManager) Add(repo formula.Repo) error {
 		return err
 	}
 
-	repo.TreeVersion = "v2"
+	repo.TreeVersion = tree.Version
```
Comment (brunasilvazup) [inline pkg/formula/repo/adder.go:66]: image

Great fix (:

File: pkg/formula/repo/adder.go [hunk 2]
```diff
@@ -116,7 +116,7 @@ func setPriority(repo formula.Repo, repos formula.Repos) formula.Repos {
 		for i := range repos {
 			r := repos[i]
 			if repo.Name == r.Name {
-				repos[i].Priority = repo.Priority
+				repos[i] = repo
```
Comment (brunasilvazup) [inline pkg/formula/repo/adder.go:119]: This change doesn't impact the movePosition function?

Comment (kaduartur) [reply pkg/formula/repo/adder.go:119]: No, it works perfectly 💯

**Review by brunasilvazup — APPROVED** [review]
:top:

File: pkg/formula/repo.go [hunk 3]
```diff
@@ -18,21 +18,35 @@ package formula
...
 import (
 	"sort"
+	"time"
 
 	"github.com/ZupIT/ritchie-cli/pkg/git"
 )
 
-const RepoCommonsName = RepoName("commons")
+const (
+	RepoCacheTime   = 24 * time.Hour
```
Comment (henriquemoraeszup) [inline pkg/formula/repo.go:27]: How about 1 hour?

Comment (kaduartur) [reply pkg/formula/repo.go:27]: In issue(#663), we decided to keep the cache for a day

**Comment by brunasilvazup** [comment]
:white_check_mark:  Tested in QA on Ubuntu, it's OK:
 - updated the existing repos with the latest_version and cache information, updated the cache when updating a repository.

**Pull request merged**